### ISSUE TITLE
Support more than one model

### ISSUE THREAD

**Issue by lingster** [ErikBjare/gptme#347]
Given the range of models that ollama can run and given that we now have smaller models that are great for specific tasks, how hard would it be for specific agents to run with a specific model. Example Qwen-coder for coding tasks, Qwen-vl for image related tasks or perhaps llama 3.3 for overall task management.

**Comment by ErikBjare** [comment]
It would be pretty easy to do such routing here: https://github.com/ErikBjare/gptme/blob/803d6e51f05d6a45de2ff82b39400143462ef2e4/gptme/llm/__init__.py#L50-L63

I'm not super interested in it though, but should be easy for gptme to modify itself to do!

**Comment by 0xbrayo** [comment]
@ErikBjare This was implemented in the last release? using `/model`

**Comment by ErikBjare** [comment]
Yes, could also be done by adding tools to outsource such reasoning: https://github.com/ErikBjare/gptme/pull/416

We are not doing any auto-routing, but I don't think we will be, at least not for now.
